nixio/exceptions/exceptions.py

```diff
@@ -77,3 +77,11 @@ class DuplicateColumnName(Exception):
     def __init__(self):
         self.message = "Column names for a DataFrame have to be unique."
         super(DuplicateColumnName, self).__init__(self.message)
+
+
+class UnsupportedLinkType(Exception):
+
+    def __init__(self, objtype, linktype):
+        self.message = "LinkType {} is not supported for {}".format(objtype,
```

Comment:
Are the arguments mix up here? Also, what is the minimal python version we are targeting? I have been using `f"LinkType {linktype}"` in other projects and that is so much nicer to read.

Reply:
Yeah, want to use f-strings everywhere too, but we're also targeting Python 2 for this release only and dropping it immediately afterwards.  It's weird to still target Py2, but 1.5 was planned for release over a year ago so it stayed in the plan since then.

Reply:
Fixed the arg order.

Reply:
Python 2, yo say. ...wow, ok. I am sorry. ;)